README.md

```diff
@@ -146,4 +146,4 @@ make prove
 ```
 
 # Contributing
-For details on how to contribute to this repo see [CONTRIBUTING](.github/CONTRIBUTING.md)
\ No newline at end of file
+For details on how to contribute to this repo see [CONTRIBUTING](.github/CONTRIBUTING.md)
```

Comment:
I have no idea what happened there, I edited the README directly in github v2 branch, but I didn't touch that, not sure why it's saying I did...